src/types.ts

```diff
@@ -10,8 +10,8 @@ export interface TableCell {
   isSelectable: boolean;
   rawRowIdx: number;
   rawColIdx: number;
-  prevQElemNumber: number;
-  nextQElemNumber: number;
+  prevQElemNumber: number | undefined;
+  nextQElemNumber: number | undefined;
```

Comment:
this is always set so should not be optional, but first and last row they can be undefined

Reply:
in theory, no difference. But sure.

Reply:
hmm well maybe that is only for function arguments it makes a difference then. anyway just a side note